components/tailscale/component.go

```diff
@@ -35,8 +34,8 @@ func New(ctx context.Context) components.Component {
 		ctx:                      cctx,
 		cancel:                   cancel,
 		checkDependencyInstalled: checkTailscaledInstalled,
-		checkServiceActive: func(ctx context.Context) (bool, error) {
-			return systemd.CheckServiceActive(ctx, "tailscaled")
+		checkServiceActiveFunc: func() (bool, error) {
+			return systemd.IsActive("tailscaled")
```

Comment:
There is an implicit dependency on dbus in IsActive... We should find another way to detect if a component is alive. 
We can add a TODO to solve later.